packages/server/src/course/course.controller.ts

```diff
@@ -113,7 +116,9 @@ export class CourseController {
     }
     await queue.save();
     // Clean up queue if necessary
-    await this.queueCleanService.cleanQueue(queue.id);
-    await this.queueSSEService.updateQueue(queue.id);
+    setTimeout(async () => {
```

Comment:
Use setTimeout so that we can return and send response to client, and do the cleaning later (1ms later)